pkg/securitypolicy/api.rego

```diff
@@ -0,0 +1,27 @@
+package api
+
+svn := "0.1.0"
+
+enforcement_points := {
+	"mount_device": {"introducedVersion": "0.1.0", "allowedByDefault": false},
+	"mount_overlay": {"introducedVersion": "0.1.0", "allowedByDefault": false},
+	"create_container": {"introducedVersion": "0.1.0", "allowedByDefault": false},
+    # the following rules are used for testing the default behavior logic. DO NOT REMOVE.
+    "__fixture_for_future_test__": {"introducedVersion": "100.0.0", "allowedByDefault": true},
```

Comment:
can we have another `api_test.rego`, which will extend the default list of `enforcement_points`, which can then be used as an override for the default embedded `apiCode` variable in the test files?

Reply:
I'm fine if we do this in a subsequent PR as long as this doesn't live too long.